example/Makefile

```diff
@@ -1,8 +1,16 @@
-compile:
+.PHONY: default run build
```

Comment:
Added some phony targets.  We are not always great about adding them (since most of the time we are just using make as a task runner), but since we are using some of Make's dependency features, I figured I would add phony targets to be more clear.

Reply:
great. Honestly I am guilty of writing makefiles like a task runner rather than a program-builder, so I forget to organize it correctly. Thanks!